### PR TITLE
[REVIEW] Corrected naming and config to match bundle

### PR DESCRIPTION
**Pull request by lucasvanlierop**
@arnovr @BauRo

### CODE REVIEW

**Comment by arnovr** [comment]
+1

File: src/DependencyInjection/DreamCommerceSingleSignOnClientExtension.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * Class DreamCommerceSingleSignOnExtension
  * @package DreamCommerce\SingleSignOnClientBundle\DependencyInjection
  */
-class DreamCommerceSingleSignOnExtension extends Extension
+class DreamCommerceSingleSignOnClientExtension extends Extension
 {
     public function load(array $configs, ContainerBuilder $container)
```
Comment (BauRo) [inline src/DependencyInjection/DreamCommerceSingleSignOnClientExtension.php]: Should't there be a docblock?

File: src/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -20,29 +20,26 @@ class Configuration implements ConfigurationInterface
...
-                        ->end()
-                        ->integerNode('token_validity')
-                            ->info('The number of seconds a token is considered valid')
-                            ->defaultValue(10)
-                        ->end()
-                            ->booleanNode('use_secure_connection')
-                            ->info('Should the handshake url be requested via https?')
-                            ->defaultValue(true);
+                ->scalarNode('server_salt')
+                    ->info('A salt configured at the DreamCommerce server')
```
Comment (BauRo) [inline src/DependencyInjection/Configuration.php]: I suggets t add a little more validation like isString
http://symfony.com/doc/2.1/components/config/definition.html#normalization

File: src/DependencyInjection/DreamCommerceSingleSignOnClientExtension.php (hunk 1 above)
Comment (lucasvanlierop) [inline src/DependencyInjection/DreamCommerceSingleSignOnClientExtension.php]: Do you mean the docblock wasn't updated, it is now.

Comment (BauRo) [inline src/DependencyInjection/DreamCommerceSingleSignOnClientExtension.php]: I mean the docblock above public function load:
/**
* @param array $config
* @param ContainerBuilder $container
*/

File: src/DependencyInjection/Configuration.php (hunk 2 above)
Comment (lucasvanlierop) [inline src/DependencyInjection/Configuration.php]: Sounds good, however as far as I can see it's not `isString()` but `ifString()`, `min()` is only available for integers, I'm open for suggestions though

File: src/DependencyInjection/DreamCommerceSingleSignOnClientExtension.php (hunk 1 above)
Comment (lucasvanlierop) [inline src/DependencyInjection/DreamCommerceSingleSignOnClientExtension.php]: Sorry, I have added that and also added the PHPCS rules for it that were not yet configured in this repo

File: src/DependencyInjection/Configuration.php (hunk 2 above)
Comment (BauRo) [inline src/DependencyInjection/Configuration.php]: You can do like:
->validate()
->ifArray()
->thenInvalid
->ifString
->then(function ($v) { validationFunction })

**Comment by BauRo** [comment]
+1

File: src/DependencyInjection/Configuration.php (hunk 2 above)
Comment (lucasvanlierop) [inline src/DependencyInjection/Configuration.php]: Check, we might do that later.

**Pull request merged**